cmd/sftp-server.go

```diff
@@ -201,6 +425,8 @@ func startSFTPServer(args []string) {
 			allowMACs = filterAlgos(arg, strings.Split(tokens[1], ","), supportedMACs)
 		case "trusted-user-ca-key":
 			userCaKeyFile = tokens[1]
+		case "disable-password-auth":
```

Comment:
```suggestion
		case "no-password-auth":
```
Just because it is shorter.

Reply:
why not just bring it down to password-auth=false

I think that works, tbh.